package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "lint": "eslint",
     "start": "webpack --config webpack.config.js"
```

Comment:
[OPTIONAL]
Ideally, the start script is to start the dev server. like:
`"start": "webpack-dev-server --mode development --open"`